src/vscode-dts/vscode.d.ts

```diff
@@ -16081,6 +16081,24 @@ declare module 'vscode' {
 		 * @returns A promise that resolves to the Debug Adapter Protocol breakpoint or `undefined`.
 		 */
 		getDebugProtocolBreakpoint(breakpoint: Breakpoint): Thenable<DebugProtocolBreakpoint | undefined>;
+
```

Comment:
Please see https://github.com/Microsoft/vscode/wiki/Extension-API-process, you can add a new proposal for this API.